reactioncommerce/admin
# feat: upgrade to be compatible with v4

Impact: **major** 
Type: **feature**

This PR updates the code to be compatible with v4.0 of admin API.

## Testing

1. Create a account using storefront.
2. Make sure to update the shopId in the .env file.
3. Run the app and try to use the login/logout functionality.
